src/realm-settings/LocalizationTab.tsx

```
@@ -115,13 +115,17 @@ export const LocalizationTab = ({
 
   useFetch(
     async () => {
-      let result = await adminClient.realms.getRealmLocalizationTexts({
-        first,
-        max,
-        realm: realm.realm!,
-        selectedLocale:
-          selectMenuLocale || getValues("defaultLocale") || whoAmI.getLocale(),
-      });
+      let result = await adminClient.realms
+        .getRealmLocalizationTexts({
+          first,
+          max,
+          realm: realm.realm!,
+          selectedLocale:
+            selectMenuLocale ||
+            getValues("defaultLocale") ||
+            whoAmI.getLocale(),
+        })
+        .catch(() => []);
```

You are swallowing the error here.  Can you just avoid the error instead?